vNext/src/ClientGenerator/BootstrapCodegen.proj

```diff
@@ -1,39 +1,79 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
```

Comment:
Hummm... I'm having a very hard time to keep old and need project definitions on the same solution. MSBuild seems doesn't like it. I've discussing with the SDK team and the only workaround was upgrade everything or downgrade everything. Anyway, was just a though.